lib/provisioner/worker/plugins/providers/fog_provider/fog_provider.json

```diff
@@ -304,6 +304,13 @@
             "override": true,
             "tip": "Zone"
           },
+          "auto_restart": {
+            "label": "Automatically restart instances on failure",
+            "type": "checkbox",
+            "default": true,
```

Comment:
not sure if this is the right default... for us internally at least, it does us no good at this time.